pkg/tag/list.go

```diff
@@ -0,0 +1,58 @@
+package tag
+
+import "github.com/reearth/reearth-backend/pkg/id"
+
+type List struct {
+	tags []id.TagID
+}
+
+func NewList() *List {
+	return &List{tags: []id.TagID{}}
+}
+
+func NewListFromTags(tags []id.TagID) *List {
+	return &List{tags: tags}
+}
+
+func (tl *List) Tags() []id.TagID {
+	return tl.tags
```

Comment:
```suggestion
	return append([]id.Tags, tl.tags...)
```

Copying prevents the slice from being modified from outside the domain model.